packages/editor/src/store/selectors.js

```diff
@@ -1701,8 +1701,8 @@ export function __experimentalGetDefaultTemplateTypes( state ) {
 export const __experimentalGetDefaultTemplatePartAreas = createSelector(
 	( state ) => {
 		const areas =
-			getEditorSettings( state )?.defaultTemplatePartAreas || [];
-		return areas?.map( ( item ) => {
+			getEditorSettings( state )?.defaultTemplatePartAreas ?? [];
```

Comment:
Agreed with the change. TBH I'm not sure why it was treated as a boolean when the relevant filter should always return an array: https://developer.wordpress.org/reference/hooks/default_wp_template_part_areas/

The nullish coalescing makes sense to me, and just to verify, I don't see any non-array usage in the plugin directory: https://wpdirectory.net/search/01HSXFV2V357M8WP7R5AA5NWGE